File: Ficheros Raspberry/programa.php
<?php

include "php_serial.class.php";

function enviar($dato){
	//exec("python programa.py $dato ");
	$serial = new phpSerial;
	$serial->deviceSet("/dev/ttyACM0");
	$serial->deviceOpen();
	sleep(1);
	$serial->sendMessage($dato);
	$serial->deviceClose();
/* $fp = fopen("/dev/ttyACM2", "w+");
   while( !$fp){
	$n = n+1;
   	$fp = fopen("/dev/ttyACM"+$n, "w+");
	echo $n;
   }
   fwrite($fp,"H");
   fclose($fp);*/
}

enviar($argv[1]);

?>

File: Ficheros Raspberry/programa.py
import serial
import sys
import time

arduino = serial.Serial('/dev/ttyACM0', 9600)
time.sleep(2)
while True:
#comando = raw_input('intoduce:')
#comando = sys.argv[1]
#print"la cadena es: ",sys.argv[1]
arduino.write("00")
time.sleep(3)
arduino.close()   
